includes/wc-stock-functions.php

```diff
@@ -43,6 +43,7 @@ function wc_update_product_stock( $product, $stock_quantity = null, $operation =
 		// Re-read product data after updating stock, then have stock status calculated and saved.
 		$product_with_stock = wc_get_product( $product_id_with_stock );
 		$product_with_stock->set_stock_status();
+		$product_with_stock->set_date_modified( date() );
```

Comment:
`Warning:  date() expects at least 1 parameter, 0 given.`

And the correct function for it is `current_time( 'timestamp', true )`.